src/test/java/uk/gov/hmcts/darts/arm/service/UnstructuredToArmBatchProcessorTest.java

```diff
@@ -166,7 +169,7 @@ void testDartsArmClientConfigInBatchQuery() {
     @Test
     void testPaginatedBatchQuery() throws IOException {
```

Comment:
Since we've touched the test here, consider renaming according to the new naming standard.